baseband/data/__init__.py

```diff
@@ -111,3 +111,10 @@ def _full_path(name, dirname=_path.dirname(_path.abspath(__file__))):
 First ten frames extracted from b0329 DRAO corrupted raw data file
 0059000.dat.
 """
+
+SAMPLE_ASP = _full_path('sample_asp.asp')
```

Comment:
The file header contains imjd and fmjd values:
```
               imjd: 53873,
               fmjd: 0.32216575686339866,
```
while the frame header seems to always have the less accurate:
```
           iMJD: 53873.0,
           fMJD: 0.3225,
```
Reading `/mnt/scratch-lustre/mahajan/Ar_P2067/P2067_1/p2067/node16/2006_05_18_01/000007.asp.raw`, however, I see that the frame header has just as accurate a fractional MJD value as the file header.  Is this the case for all ASP files, and if so, can the sample file be altered to match?